themes/classic/templates/catalog/_partials/miniatures/product.tpl

```diff
@@ -49,7 +49,9 @@
                 <span class="sr-only">{l s='Regular price' d='Shop.Theme.Catalog'}</span>
                 <span class="regular-price">{$product.regular_price}</span>
                 {if $product.discount_type === 'percentage'}
-                  <span class="discount-percentage">{$product.discount_percentage}</span>
+                  <span class="discount-percentage discount-product">{$product.discount_percentage}</span>
+                {elseif $product.discount_type === 'amount'}
+                  <span class="discount-amount discount-product">-{$product.discount_amount}</span>
```

Comment:
Could you please remove this "calculation" from the template to do it in the presenter like we do it for the percentage discount here: https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Product/ProductPresenter.php#L159